finals-2024-01-16/code/5089-jovan-karajanov/src/app/genres/genre/genre.component.html

```diff
@@ -0,0 +1,7 @@
+<div>
+    <h2>Genres</h2>
+    <ul>
+      <li *ngFor="let genre of genres">{{ genre }}</li>
```

Comment:
There is a minimal error here that prevents the app building. +3 points for app build status